pkg/config/config.go

```diff
@@ -33,25 +35,62 @@ func getConfigHomeDir() string {
 }
 
 func LoadFile() error {
-	viper.SetConfigFile(viper.GetString(ArgConfig))
-	err := viper.ReadInConfig()
-	if err != nil {
-		return err
+	path := viper.GetString(ArgConfig)
+	if !filepath.IsAbs(path) {
+		path, _ = filepath.Abs(path)
 	}
-	return nil
+	fileInfo, statErr := os.Stat(path)
+	if statErr != nil {
+		return statErr
+	}
+
+	perm := fileInfo.Mode().Perm()
+	permNumberBase10 := int64(perm)
+	strBase10 := strconv.FormatInt(permNumberBase10, 8)
+	permNumber, _ := strconv.Atoi(strBase10)
+
+	//TODO: Recheck if keeping implementation below
+	system := runtime.GOOS
+	if system == "windows" {
+		if permNumber == int(666) {
+			viper.SetConfigFile(viper.GetString(ArgConfig))
+			err := viper.ReadInConfig()
+			if err != nil {
+				return err
+			}
+			return nil
+		} else {
+			fmt.Printf("perm: %v", permNumber)
```

Comment:
Please remove the fmt.Printf in here and below, eventually, you can add it to the `errors.New("no permission for the config file, expected 600, got "+permNumber)` to be more explicit, but remaining this way will affect the json/text format printing of the tool.

Reply:
done